### ISSUE TITLE
Allow await examples without a wrapping async function

### ISSUE THREAD

**Issue by gr2m** [zeke/standard-markdown#31]
Browsers have `await` in the global space so it would be nice when showing examples in the README to allow something like this

```js
const result = await octokitRequest('GET /orgs/:org/repos', {
  headers: {
    authorization: 'token 0000000000000000000000000000000000000001'
  },
  org: 'octokit',
  type: 'private'
})
```

I looked for an option to add the package.json but no luck. Is there a way to workaround it or could that even be allowed by default?

**Comment by zeke** [comment]
Yeah let's add support for that by default. Is there an eslint rule for it?

**Comment by zeke** [comment]
Comments on this issue seem to be disappearing (at least for me)... emailing support@github.com.

**Comment by gr2m** [comment]
I’ve researched a bit but couldn’t find any rule. I’m not very familiar with eslint, do you know someone we could ask for help?

**Comment by feross** [comment]
Hey folks! Sorry for the delay!! (I was taking a much-needed break from OSS). I totally agree that top-level `await` in readme docs is great!

I think that for top-level `await` support, the most obvious route would be to wait for ESLint to support it. They have a policy of supporting any JS feature that is at least Stage 4. Currently top-level `await` is Stage 3. https://github.com/tc39/proposal-top-level-await

But maybe if you're using babel with the `babel-eslint` parser already, there's a transform that you could install to handle this?
